airbyte-integrations/connectors/source-google-analytics-v4/metadata.yaml

```diff
@@ -29,9 +29,9 @@ data:
     cloud:
       enabled: false
     oss:
-      enabled: true
+      enabled: false
   releaseStage: generally_available
-  supportLevel: certified
+  supportLevel: community
```

Comment:
shouldn't we set the supportLevel to archived [like we did with other connectors we archived](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-appstore-singer/metadata.yaml#L29)?